modules/gridmap/grid_map_editor_plugin.cpp

```diff
@@ -99,6 +99,19 @@ void GridMapEditor::_menu_option(int p_option) {
 				int idx = options->get_popup()->get_item_index(MENU_OPTION_X_AXIS + i);
 				options->get_popup()->set_item_checked(idx, i == new_axis);
 			}
+			if (edit_axis == Vector3::AXIS_Y && new_axis != edit_axis) {
+				int item1 = options->get_popup()->get_item_idx_from_text(TTR("Next Floor"));
```

Comment:
I think it's better to use `get_item_index` and these enums:
```
MENU_OPTION_NEXT_LEVEL,
MENU_OPTION_PREV_LEVEL,
```

Reply:
I missed them. I'll make the changes ASAP. Should I be amending the commit or should I start a new pull request afterwards?

Reply:
Please amend the commit if possible. While doing so, please also remove the unrelated commit (ef24984).

(I'm not good with git and not sure about the commands.)

Reply:
All commands are in http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html

Reply:
I think I finally managed to do it. I hope I get better fast.